.github/workflows/docker_build_lite.yml

```diff
@@ -21,7 +21,7 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        branch: [ latest, mysql57, mysql80, percona57, percona80 ]
+        branch: [ latest, mysql80, percona80 ]
```

Comment:
Is `latest` the same as `mysql80`? Same image with two tags?

Reply:
That's correct, we have a static link between `./docker/lite/Dockerfile` and `./docker/lite/Dockerfile.mysql80`, they're exactly the same Dockerfile. But for the lite image, we do explicitly push three different tags:

```
# For a tag/release:

vitess/lite:v19.0.6
vitess/lite:v19.0.6-mysql80
vitess/lite:v19.0.6-percona80
```

```
# For a commit to the main branch, only without the tag without suffix:

vitess/lite:v19.0.6
```

This behavior has been correct in v20+, we stopped publishing the extra `...-mysql80` image as it became irrelevant.